source/fe/fe_dgq.cc

```diff
@@ -656,10 +657,17 @@ FE_DGQ<dim, spacedim>::memory_consumption () const
 
 
 
+// ------------------------------ FE_DGQArbitraryNodes -----------------------
+
 template <int dim, int spacedim>
 FE_DGQArbitraryNodes<dim,spacedim>::FE_DGQArbitraryNodes (const Quadrature<1> &points)
-  : FE_DGQ<dim,spacedim>(points)
-{}
+  : FE_DGQ<dim,spacedim>(Polynomials::generate_complete_Lagrange_basis(points.get_points()))
```

Comment:
I'm ok with just making the change without any note or deprecation. The use case is indeed pretty narrow.

I would also be ok if you wanted to just have both constructors. The code duplication is not large. 

Your choice -- do as you think is most appropriate.